edb/server/server.py

```diff
@@ -535,6 +539,34 @@ async def reload_sys_config(self):
         self._dbindex.update_sys_config(cfg)
         self._reinit_idle_gc_collector()
 
+    def schedule_reported_config_if_needed(self, setting_name):
+        setting = self._config_settings[setting_name]
+        if setting.report:
+            self.create_task(
+                self.load_reported_config(), interruptable=True)
+
+    def get_report_config_data(self) -> bytes:
+        return self._report_config_data
+
+    async def load_reported_config(self):
+        syscon = await self._acquire_sys_pgcon()
```

Comment:
This seems to only reload the config.  I can't seem to find where it sends the updated config to the clients.

Reply:
Yes, we'll need to fix this in RC3.